### PR TITLE
[Cherry 3.4.x] test: test_access.py and test_compat fixes

### PR DESCRIPTION
**Pull request by merlin-northern**

### CODE REVIEW

**Comment by merlin-northern** [comment]
@merlin-northern start pipeline please honey

**Comment by merlin-northern** [comment]
> @merlin-northern start pipeline please honey

wow @lluiscampos check it out.

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline pretty please

**Comment by merlin-northern** [comment]
@lluiscampos only well known failures here.

File: extra/integration-testing/test-compat/docker-compose.compat-mender-3.4.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  mender-client-3-4:
+    image: mendersoftware/mender-client-qemu:mender-3.4
+    networks:
+       - mender
+    stdin_open: true
+    tty: true
+    privileged: true
+    environment:
+      TENANT_TOKEN: ""
+
```
Comment (lluiscampos) [inline extra/integration-testing/test-compat/docker-compose.compat-mender-3.4.yml:12]: Technically speaking this one is not necessary here, as this is the version that is tested in every other test. But it is harmless so :ok_hand:

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**